lib/generators/sufia/install_generator.rb

```diff
@@ -172,8 +172,8 @@ def inject_routes
         "  # This behavior seems to show up only in production mode.\n" \
         "  mount Sufia::Engine => '/'\n"
 
-      sentinel = /devise_for :users/
-      inject_into_file 'config/routes.rb', routing_code, after: sentinel, verbose: false
+      sentinel = /end\Z/
```

Comment:
The `\Z` matches the end of string (effectively EOF). So I'm inserting before the last `end` in the file.